src/tool/xmeta/cpp/models/property_model.h

```diff
@@ -57,9 +69,38 @@ namespace xlang::xmeta
             return m_set_method;
         }
 
+        auto const& get_semantic() const noexcept
+        {
+            return m_semantic;
+        }
+
+        void create_get_method()
+        {
+            m_get_method = std::make_shared<method_model>(
```

Comment:
The getter and setter are owned by the parent interface/class, not by this property. In other words, they need to be created on the owning interface/class, and the property holds a non-owning reference to the getter and/or setter method.